js/classes.js

```diff
@@ -96,8 +96,35 @@ class Projectile {
     this.position.x += this.velocity.x;
     this.position.y += this.velocity.y;
   }
+  static createProjectile(xVelocity = 0, yVelocity = -10, color = "red") {
```

Comment:
I don't want to be annoying but relooking on this code, I think better place for **createProjectile** would be as method of **player** because that would solve the fact we are using 'player' global variable. (If this was method of player, we would have player in class fields and we could avoid using global variable)

Reply:
I won't delay the pull request for this. Just fix the other comment and we can merge if you want

Reply:
This project has serious issue of using global variables and it is very unsettling. This + adding typescript is the first thing i would want to solve if we want to continue with this project